Community.md

```diff
@@ -0,0 +1,47 @@
+# Community Continuity Guidelines v2.0.0
+
+As the creators, and maintainers of this project. We want to ensure that the project lives and continues to grow. Not blocked by any singular person's computer time.
+
+One of the simplest ways of doing this, is by encouraging a larger set of shallow contributors.
+
+#### Ownership
+
+If you get a merged PR, regardless of content (typos, code, doc fixes) then you're eligible for push access to this organization. We check for this on PR merges and extend an invite via GitHub.
+
+Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project. If you are working with a codebase, then you probably have the skills to improve it, and offer valuable feedback.
```

Comment:
I like "offhand" to describe this reaction; it's a common refrain we hear and using this phrasing frames it in a way that suggests its a reasonable feeling, even if the concern is unwarranted. I think it will help convince those on-the-fence to try this method of doing things.